src/services/person/person.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { rest } from 'msw';
+import { getPersonById } from './person';
+import { config } from '../config';
+import { server, mockPerson } from '../../mocks';
+
+test('it throws an error if the id is falsy', async () => {
+    await expect(() => getPersonById({ id: '' })).rejects.toThrow(
+        'Missing `id` for Person'
+    );
+});
+
+test('it returns data on an ok response', async () => {
+    const data = await getPersonById({
+        id: '1234',
+    });
+    expect(data).toStrictEqual(mockPerson);
+});
+
+test('it throws an ResponseException on bad request', async () => {
+    const url = `${config.personApiUrl}/persons/1234`;
+    server.use(
```

Comment:
move this to test-utils as utility helpers like we have done else where ie, `get`, `post`, `patch`

Reply:
moved to test-utils in the same way as MFEs